src/ansys/dpf/post/modal_mechanical_simulation.py

```diff
@@ -77,6 +79,13 @@ def _get_result(
                 List of IDs of elements to get results for.
             named_selections:
                 Named selection or list of named selections to get results for.
+            expand_cyclic:
+                For cyclic problems, whether to expand the sectors.
+                Can take a list of sector numbers to select specific sectors to expand.
+                If the problem is multi-stage, can take a list of lists of sector numbers, ordered
+                by stage.
+            phase_angle_cyclic:
+                For cyclic problems, phase angle to apply.
```

Comment:
maybe add the angle unit here

Reply:
andd sectors number start at 0 or 1?

Reply:
@cbellot000 good idea for the angle, and yes, the "base_sector" label does start at `1`.